src/azul/__init__.py

```diff
@@ -129,11 +129,20 @@ def unqualified_resource_name_or_none(self, qualified_resource_name: str, suffix
     def subdomain(self, lambda_name):
         return os.environ['AZUL_SUBDOMAIN_TEMPLATE'].format(lambda_name=lambda_name)
 
-    def api_lambda_domain(self, lambda_name):
-        return config.subdomain(lambda_name) + "." + config.domain_name
+    def api_lambda_domain(self, lambda_name: str) -> str:
+        return self.subdomain(lambda_name) + "." + self.domain_name
 
-    def service_endpoint(self):
-        return "https://" + config.api_lambda_domain('service')
+    def lambda_endpoint(self, lambda_name: str) -> str:
+        return "https://" + self.api_lambda_domain(lambda_name)
+
+    def indexer_endpoint(self) -> str:
```

Comment:
There are two places that would benefit from using this method:

https://github.com/DataBiosphere/azul/search?q=%22api_lambda_domain+indexer%22&unscoped_q=%22api_lambda_domain+indexer%22